CONTRIBUTING.md

```diff
@@ -86,12 +86,47 @@ A lot of the instructions below replicate what the [PR CI pipeline] is doing.
 1. You are now ready to submit your PR.
 1. After your PR ise merged, most likely you will want to read [`How to deploy your changes`](#how-to-deploy-your-changes) to verify they got deployed.
 
-# New linter rule promotion strategy
+# (WIP) New linter rule promotion strategy
 
-1. Merge new rules to main. As a result, they show up in the Staging LintDiff pipeline.
+1. __MAYBE__: run the new rule (only the new rule) locally for all of the existing swagger specs and aggregate the results
+1. Merge new rules to main
+    1. As a result, they show up in the Staging LintDiff pipeline.
```

Comment:
We should link here to https://github.com/Azure/azure-openapi-validator/blob/main/CONTRIBUTING.md#deploy-to-staging-lintdiff to clarify how to verify it actually got deployed